### ISSUE TITLE
Arms of V4 delaminate

### ISSUE THREAD

**Issue by cwei** [tbs-trappy/source_one#24]
After a baby crash into a field (mud) the front arm is delaminated like puff pastry.
Nearly every layer of carbon came off.

Is it a quality problem of the carbon or are the 4mm too weak?
Or both?

I had no issues with the V3.

**Comment by cwei** [comment]
Here is another example https://www.youtube.com/watch?v=Xlhkx9ak-zo

**Comment by cwei** [comment]
Replaced the original 4 mm arms with 5 mm arms. (thanks Kai for cutting)
Now the quad flies like a charm.
No torsion and bending of the arms any more.
The vibrations and oscillations are gone.

**Comment by krunked-fpv** [comment]
yeah i have a 5mm version and 6mm version -- 6mm version flies best imo :)

**Comment by cwei** [comment]
This cool [frame resonance analysis](https://www.youtube.com/watch?v=-W7RzZ66eRo) of the S1 V4 shows why it flies better with thicker/stiffer arms.
